views/index.ejs

```diff
@@ -21,12 +21,21 @@
 <body>
   <div id="loader">Please wait, loading...</div>
   <div class="header">
-    <h2 id="title"><%= title %></h2>
+    <h2 id="title">
+      <%= title %>
+    </h2>
   </div>
   <div class="meter">
     <div id="emojis" class="emojis"></div>
     <input min="0" step="0.1" id="slider" class="slider" type="range" />
   </div>
+  <script>
+    let root = document.documentElement;
+    root.style.setProperty("--bg-color", "<%=bgColor%>");
+    root.style.setProperty("--text-color", "<%=textColor%>");
+    root.style.setProperty("--slider-thumb-color", "<%=sliderThumbColor%>");
+    root.style.setProperty("--slider-bg-color", "<%=sliderBgColor%>");
+  </script>
```

Comment:
I am not sure if this is the best way to do it. I have never used eJs before, there isn't another way to change the css with eJs?